README.md

```diff
@@ -252,6 +255,21 @@ Linux default:
 Windows default:
 `sleep 5 & ipconfig`
 
+## Post Create Script
+
+This offers a way to make VMs ready for connections by a Test Kitchen transport. In some cases, the default firewall settings will not allow immediate
+connections via WinRM. With a post create script, you can remotely execute statements to enable this.
+
+Add all necessary commands with the `post_create_script` property. These commands get invoked via the Guest Operations API (VMware Tools), which does not
+require network connectivity. As such, they need additional privileges in vCenter.
```

Comment:
```suggestion
The Guest Operations API (VMware Tools) executes the commands from the `post_create_script`. The GuestOperatons endpoint does not require network connectivity, but it will require additional privileges in vCenter.
```